source/common/common/backoff_strategy.h

```diff
@@ -17,9 +17,9 @@ class JitteredBackOffStrategy : public BackOffStrategy {
 
 public:
   /**
-   * Use this constructor if max_interval need to be enforced.
```

Comment:
@htuch sorry. missed this earlier. This comment does not make sense now that we have removed the other constructor that used to allow this object creation without bounds